pkg/flow/services/accounts.go

```diff
@@ -0,0 +1,458 @@
+	}
+}
+
+// Get gets an account based on address
+func (a *Accounts) Get(address string) (*flowsdk.Account, error) {
+	a.logger.StartProgress(fmt.Sprintf("Loading %s...", address))
+
+	flowAddress := flowsdk.HexToAddress(
+		strings.ReplaceAll(address, "0x", ""),
+	)
+
+	account, err := a.gateway.GetAccount(flowAddress)
+	a.logger.StopProgress("")
+
+	return account, err
+}
+
+func (a *Accounts) Add(
+	name string,
+	accountAddress string,
+	signatureAlgorithm string,
+	hashingAlgorithm string,
+	keyIndex int,
+	keyHex string,
+	keyContext string,
+	overwrite bool,
+	path []string,
+) (*flow.Account, error) {
+
+	existingAccount := a.project.GetAccountByName(name)
+	if existingAccount != nil && !overwrite {
+		return nil, fmt.Errorf("account with name [%s] already exists in the config, use --overwrite flag if you want to overwrite it", name)
+	}
+
+	sigAlgo, hashAlgo, err := util.ConvertSigAndHashAlgo(signatureAlgorithm, hashingAlgorithm)
+	if err != nil {
+		return nil, err
+	}
+
+	if keyIndex < 0 {
+		return nil, fmt.Errorf("key index must be positive number")
+	}
+
+	address := flowsdk.HexToAddress(accountAddress)
+	chainID, err := util.GetAddressNetwork(address)
+	if err != nil {
+		return nil, err
+	}
+
+	confAccount := config.Account{
+		Name:    name,
+		Address: address,
+		ChainID: chainID,
+	}
+
+	accountKey := config.AccountKey{
+		Index:    keyIndex,
+		SigAlgo:  sigAlgo,
+		HashAlgo: hashAlgo,
+	}
+
+	// hex key
+	if keyHex != "" {
+		_, err := crypto.DecodePrivateKeyHex(sigAlgo, keyHex)
+		if err != nil {
+			return nil, fmt.Errorf("key hex could not be parsed")
+		}
+
+		accountKey.Type = config.KeyTypeHex
+		accountKey.Context = make(map[string]string)
+		accountKey.Context[config.PrivateKeyField] = keyHex
+
+	} else if keyContext != "" {
+		keyCtx, err := keys.KeyContextFromKMSResourceID(keyContext)
+		if err != nil {
+			return nil, fmt.Errorf("key context could not be parsed %s", keyContext)
+		}
+
+		accountKey.Type = config.KeyTypeGoogleKMS
+		accountKey.Context = keyCtx
+
+	} else {
+		return nil, fmt.Errorf("either --privatekey or --context flag must be provided")
+	}
+
+	confAccount.Keys = []config.AccountKey{accountKey}
+
+	account, err := flow.AccountFromConfig(confAccount)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = account.DefaultKey().Signer().Sign([]byte("test"))
+	if err != nil {
+		return nil, fmt.Errorf("could not sign with the new key")
+	}
+
+	a.project.AddOrUpdateAccount(account)
+
+	err = a.project.Save(path[0]) // only allow saving to one config for now
+	if err != nil {
+		return nil, err
+	}
+
+	a.logger.Info("Account added to configuration\n")
+
+	return account, nil
+}
+
+// StakingInfo gets staking info for the account
+func (a *Accounts) StakingInfo(accountAddress string) (*cadence.Value, *cadence.Value, error) {
+	a.logger.StartProgress(fmt.Sprintf("Fetching info for %s...", accountAddress))
+
+	address := flowsdk.HexToAddress(
+		strings.ReplaceAll(accountAddress, "0x", ""),
```

Comment:
I don't think this is needed anymore -- the SDK should automatically handle hex prefixes

Reply:
ohh, that got implemented? awesome :))) I will remove this from everywhere